### ISSUE TITLE
Stay informed of application status: Update for VHA and CHAMPVA forms

### ISSUE THREAD

**Issue by LoisLewisUSDS** [department-of-veterans-affairs/vets-design-system-documentation#3398]
### Duplicate check

- [x] I've searched for any related issues and avoided creating a duplicate issue.

### This update is for:

Pattern

### What is the name?

Stay informed of application status

### What is the nature of this update?

- [ ] How to build this component/pattern
- [ ] When to use this component/pattern
- [ ] When to use something else
- [ ] Usability guidance
- [ ] Accessibility considerations
- [x] Content considerations
- [ ] Implementation
- [ ] Package information
- [x] Addition to Forms Library documentation
- [ ] Update to existing Forms Library documentation

### What problem does this solve?

This pattern will need to be used by healthcare applications for veterans and beneficiaries and claims for champva and FMP reimbursement. It seems to only address benefits documents.

**Comment by pshahVA** [comment]
We don't have the capability to get the application status yet, but due to some recent escalation we may implement something over the next few months. @aklausmeier is looking at the design.

**Comment by aklausmeier** [comment]
> Are you following the same pattern as Benefits forms?

@humancompanion-usds Yes, we will likely follow this pattern. Travel Pay implemented a similar pattern to CST as well.

**Comment by humancompanion-usds** [comment]
Okay, I'm going to close this then. @aklausmeier - If you do end up deviating from the pattern or wish for us to make some sort of change/update to it to accommodate CHAMPVA then file an experimental design request to change the pattern. But given that there isn't anything for us to do here at the moment I will close this.
